shared/navLinks/links.ts

```diff
@@ -98,26 +98,20 @@ export const links = [
     ],
   },
   {
-    label: messages.company,
-    color: "pink",
+    label: messages.developers,
```

Comment:
This appears twice in the navigation bar - did we intentionally want to get rid of the company section here?